benefits/working_hours.md

```diff
@@ -5,9 +5,10 @@ We care about the quality of the work we produce rather than the amount of hours
 Some things to note:
 
  - We do not have a set amount of hours you need to work in a day or a week. Some people will work 6 hours, some 7, sometimes more than 8. Many of us will vary our hours day to day whilst others prefer routine.
- - We do not have a specific start time but we do have our All Hands three times a week at 0950 so you should get into the office or dial in before that time.
+ - We do not have a specific start time but we do have our All Hands three times a week at 0950 so you should get into the office or dial in before that time in our #general slack channel.
  - Likewise there isn't a time we expect everyone to leave the office. Of course if you're leaving at 1630 every day and are having problems delivering on your commitments someone will likely provide you with [feedback](https://github.com/madetech/handbook/blob/working_hours/policies/continuous_feedback.md) to help improve things.
  - Pressure can creep in around deadlines and sometimes staying late to ensure a smooth launch or to help diagnose and fix a problem is a necessity. However these occassions should be very rare. If you regularly feel this pressure, say more than twice a year then speak to a colleague.
  - Pressure can also arise when peers work seemingly longer hours than yourself. All we can suggest is that we have open and honest ways of providing feedback so unless someone has raised an issue with you, you're likely doing fine.
  - We need to respect our commitments so if we have a meeting booked for a certain time you are expected to be accommodating. This isn't to say you can't be responsible for setting meeting times that fit your working style better.
  - If you are taking a half day, read about our [untracked holiday](https://github.com/madetech/handbook/blob/master/benefits/untracked_holiday.md)
+ - It's worth noting most of our employment contracts with staff do in fact have set contractual hours. This is so we can withdraw this privilege if necessary.
```

Comment:
Perhaps "This is so we can withdraw this privilege if necessary." could be written as "This privilege can be withdrawn as an outcome of continuous feedback." I would want to get optimistic wording in there, rather than it sounding pessimistic!